src/app/launcher/launcher.component.ts

```diff
@@ -29,6 +29,11 @@ export class LauncherComponent implements AfterViewInit, OnInit {
    */
   @Input() importApp: boolean = false;
 
+  /**
+   * Setting the flow to 'launch' will skip the pipeline step and show a cluster dropdown. Defaults to 'osio'.
+   */
+  @Input() flow: string = 'osio';
```

Comment:
I had added targetEnvironment to distinguish between the OSIO and Launcher flows. In OSIO, we obviously set this to 'osio' to hide certain steps.

I'm thinking that we probably don't need two properties that control the flow. That is, one or the other should do?